CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+# Contributing to docker-node
+
+Thank you for your contribution. Here are a set of guidelines for contributing to the docker-node project.
+
+## Adding dependencies to the base images
+
+NodeJS is a big ecosystem with a variety of different use cases. The docker images for node are designed to provide the minimum for running code node.  Additional dependencies (including dependencies for npm or yarn such as git) will not be included in these base images and will need to be included in descendent image. We however maintain a list of common dependencies you can add to your image (TODO). Feel free to open a PR to document more of them.
```

Comment:
There's no rush to merging this. We'll see if anyone has a better idea on how to word this. I'll try to get started on documenting the common dependencies soon. I will do git for alpine, node-gyp, and the other one we decided not to merge not too long ago.